h/templates/activity/search.html.jinja2

```diff
@@ -36,11 +36,18 @@ Users (first ten):
     <li>
       <p><strong>{{ timeframe.label }}</strong></p>
       <ol>
-        {% for document, results in timeframe.document_buckets.items() %}
+        {% for _, bucket in timeframe.document_buckets.items() %}
```

Comment:
Tiny thing, but `for bucket in timeframe.document_buckets.values()` would have the same effect and you wouldn't have to discard the documents.